### PR TITLE
Complete HW 2 from 2/2/15 class. 

### PR DESCRIPTION
**Pull request by blackeaglejs**
Comfort Level: 4

### CODE REVIEW

**Comment by blackeaglejs** [comment]
@mobjohnson Please see above.

File: class03/zoheb/disney.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+ruby_characters = ["Mickey Mouse", "Aladdin", "Snow White", "Peter Pan", "Simba", "Mulan", "Goofy", "Elsa", "Tarzan", "Donald Duck", "Shere Khan", "Prince Charming", "Cinderella", "Mufasa", "Belle", "Jiminy Cricket"]
+
+puts ruby_characters.sample + " \n"
+
```
Comment (mobjohnson) [inline class03/zoheb/disney.rb]: A puts includes the "\n" already

File: class03/zoheb/wines.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  {:label => "Markham", :type => "Chardonnay", :color => "white"},
+  {:label => "Angeline", :type => "Pinot Noir", :color => "red"}
+]
+
+# Adds a wine of choice to the cellar 
+ruby_wine_cellar.push({:label => "Richebourg", :type => "Pinot Noir", :color => "red"})
+
+# Returns a random wine from the cellar. 
+puts "This is a random wine."
+puts ruby_wine_cellar.sample
```
Comment (mobjohnson) [inline class03/zoheb/wines.rb]: This ends up printing all of the wines

**Comment by mobjohnson** [comment]
Good Job! I added some comments to your code, but you need not resubmit.

**Pull request merged**